lib/trino-plugin-toolkit/src/main/java/io/trino/plugin/base/security/LocalFileBasedSystemAccessControlModule.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.base.security;
+
+import com.google.inject.Binder;
+import com.google.inject.Inject;
+import com.google.inject.Provides;
+import io.airlift.configuration.AbstractConfigurationAwareModule;
+import io.airlift.configuration.ConfigurationFactory;
+import io.airlift.log.Logger;
+import io.trino.spi.security.SystemAccessControl;
+
+import java.nio.file.Paths;
+import java.util.Map;
+
+import static com.google.common.base.Suppliers.memoizeWithExpiration;
+import static io.airlift.configuration.ConfigBinder.configBinder;
+import static io.trino.plugin.base.util.JsonUtils.parseJson;
+import static java.util.concurrent.TimeUnit.MILLISECONDS;
+
+public class LocalFileBasedSystemAccessControlModule
+        extends AbstractConfigurationAwareModule
+{
+    private static final Logger log = Logger.get(LocalFileBasedSystemAccessControlModule.class);
+
+    public LocalFileBasedSystemAccessControlModule()
+    {
+        super();
+    }
+
+    public LocalFileBasedSystemAccessControlModule(Map<String, String> config)
+    {
+        super();
+        this.setConfigurationFactory(new ConfigurationFactory(config));
+    }
```

Comment:
This is not required. Ideally if the  parentModule which installs this `LocalFileBasedSystemAccessControlModule` is an extenstion of `AbstractConfigurationAwareModule` - then the configuration property will be propogated here.

Reply:
you are right. Removed it